changelog/unreleased/issue-2688

```diff
@@ -0,0 +1,6 @@
+Bugfix: Handle comma-separated list tags when using backup command
+
+Creating a backup by specifying comma-separated tags created a backup with this concatenated list as a tag.
+For example `restic backup --tag foo,bar` creates a new snapshot with one "foo,bar" tag instead of two tags, one "foo" and one "bar". Fixed now.
+
+https://github.com/restic/restic/issues/2688
```

Comment:
Please update the entire changelog entry to the following :)

~~~
Bugfix: Make backup and tag commands separate tags by comma

Running `restic backup --tag foo,bar` previously created snapshots with one
single tag containing a comma ("foo,bar") instead of two tags ("foo", "bar").
Similarly, the `tag` command's --set, --add and --remove options would treat
"foo,bar" as one tag instead of two tags. This was inconsistent with other
commands and often unexpected when one intended "foo,bar" to mean two tags.

To be consistent in all commands, restic now interprets "foo,bar" to mean two
separate tags ("foo" and "bar") instead of one tag ("foo,bar") everywhere,
including in the `backup` and `tag` commands.

NOTE: This change might result in unexpected behavior in cases where you use
the `forget` command and filter on tags like "foo,bar". Snapshots previously
backed up with `--tag foo,bar` will still not match that filter, but snapshots
saved from now on will match that filter.

To replace "foo,bar" tags with "foo" and "bar" tags in old snapshots, you can
first generate a list of the relevant snapshots using a command like
`restic snapshots --json --quiet | jq '.[] | select(contains({tags: ["foo,bar"]})) | .id'`
, and then use `restic tag --set foo --set bar snapshotID [...]` to set the new
tags. Please adjust the commands to real tag names and any additional tags.

https://github.com/restic/restic/issues/2688
~~~

Reply:
Is this the only this you want ? 😄

Reply:
Yep, it is :D

Reply:
@SkYNewZ *gentle ping* - I just popped back here to remind myself exactly what the problem is/whether fixed (i.e. checking that `backup --tag a --tag b` should work) - it'd be nice to have this in a release, thanks for your work on it :slightly_smiling_face:

Reply:
Yes, I'm sorry, I didn't take the time to make the changes in the changelog. I'll take care of it right now

Reply:
After rebasing my changes, I update the changelog entry as you asked to